apps/api/src/lessons/adminLessons.service.ts

```diff
@@ -216,7 +225,7 @@ export class AdminLessonsService {
       conditions.push(eq(lessons.archived, filters.archived));
     }
 
-    return conditions.length ? conditions : [sql`1=1`];
+    return conditions;
```

Comment:
Will it work in the queries when it returns an empty array instead of a 1=1 placeholder?

Reply:
NIce try,  Didn't think of that, In documentation is 
![image](https://github.com/user-attachments/assets/631fdb1a-5114-4685-a959-f5a6fb700a44)

Reply:
But in this case, we can return empty array which get error in query executing.